### PR TITLE
Add routing between pages

### PR DESCRIPTION
**Pull request by jagjennings**
- moved navbar component directly into app component
- added routerlinks for create page and result page
- route to result page on create rss feed button click

### CODE REVIEW

**Review by ERetana2 — DISMISSED** [review]

**Review by ERetana2 — DISMISSED** [review]

**Review by qihuang2 — DISMISSED** [review]

File: launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.center-page {
+    left: 50%;
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css:2]: Style nit:  Can we either put attributes that start with "-" either all at the end of the block or at the top and always keep it that way?

File: launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    top: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.icons{
+    padding-right: 10px;
+}
+
+.logo{
+    height:50px;
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css]: For consistency sake, can we always add a space after the colon? :)

File: launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.html [hunk 3]
```diff
@@ -2,17 +2,17 @@
...
+        <input matInput name="title" placeholder="Ex. This American Life">
     </mat-form-field>
     <br />
 
     <h2>Provide Your mp3 Link</h2>
     <mat-form-field appearance="outline">
         <mat-label>mp3 Link</mat-label>
-        <input matInput placeholder="Paste the mp3 link here">
+        <input matInput name="mp3Link" placeholder="Paste the mp3 link here">
     </mat-form-field>
```
Comment (bramanudom) [inline launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.html:13]: Just for my knowledge, is it always an mp3? 

Super nit: If not we can name the input something more generic like sourceFileUrl or something like that. 

yes, this input is always going to be a link to an mp3 file.

**Review by bramanudom — DISMISSED** [review]

File: launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css (hunk 1 above)
Comment (jagjennings) [reply launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css:2]: done.

File: launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css (hunk 2 above)
Comment (jagjennings) [reply launchpod/src/main/webapp/angular-launchpod/src/app/app.component.css]: done.

File: launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.html (hunk 3 above)
Comment (jagjennings) [reply launchpod/src/main/webapp/angular-launchpod/src/app/create-form/create-form.component.html:13]: yes, this input is always going to be a link to an mp3 file.

**Review by kassenq — APPROVED** [review]

**Review by bramanudom — APPROVED** [review]

**Review by ERetana2 — APPROVED** [review]

**Pull request merged**